### ISSUE TITLE
Hyperdeck

### ISSUE THREAD

**Issue by kraves-oz** [Alteka/VTKounter#21]
Quick question over last line of /src/vtApps/Hyperdeck.js

currently:
`module.exports = vtAppMitti`

Should it be ... 
`module.exports = vtAppHyperdeck`

**Comment by DrewLX** [comment]
Yeah the Hyperdeck module is far from working. It's a holding file at best because whenever I have the time to develop it I don't have a hyperdeck with me.
